spec.md

```diff
@@ -388,13 +388,34 @@ both appear within the same JSON object.
   or application protocol's content-type metadata property. Normative rules
   for the binary mode and the content-type metadata mapping can be found 
   in the respective transport mapping specifications.
-    
+  
 * Constraints:
   * OPTIONAL
   * If present, MUST adhere to the format specified in
     [RFC 2046](https://tools.ietf.org/html/rfc2046)
 * For Media Type examples see [IANA Media Types](http://www.iana.org/assignments/media-types/media-types.xhtml)
 
+### application-properties
+* Type: `Map`
+* Description: A list of "key-value" pairs providing information about the
+  event, which are used by an event-based application. The "key-value" pairs
+  MUST be in a flat structure. The value MUST be a string. There MUST be no
+  duplication of keys. One usage example: an application workflow can use a
+  "key-value" pair to correlate multiple events (from different event sources)
+  associated with a serverless application workflow.
+* Constraints:
+  * REQUIRED
+* Examples:
+``` JSON
+  application-properties
+  {
+     "building address": "12 Main Street, Copenhagen, Denmark, 123456",
```

Comment:
I would suggest that we don't allow spaces in the key names (just alphanumeric) otherwise serializing these as http headers will be tricky.

Reply:
Doug, any other restriction on the key names?